src/controllers/raw/rawHTMLToPDF.js

```diff
@@ -4,17 +4,23 @@ import uuid from 'uuid/v4';
 
 export default async ( req, res ) => {
   /* Get the html body */
-  const { html, sendFile } = req.body;
+  const { html, sendFile, urlToPDF } = req.body;
```

Comment:
I think just `url` would be a better field name. Thoughts?

Reply:
ok will update that